Modules/_threadmodule.c

```diff
@@ -2594,6 +2648,31 @@ thread_module_exec(PyObject *module)
     }
 #endif
 
+#ifdef MS_WINDOWS
+    HMODULE kernelbase = GetModuleHandleW(L"kernelbase.dll");
+    if (kernelbase != NULL) {
```

Comment:
If this check fails, we likely want to report a system error since everything is broken, but at least we should call `DelAttr` to remove the functions, since the function pointers will be null.

Reply:
`set_name()`/`_get_name()` is a minor feature of the `_thread` module, I would prefer to not prevent to import `_thread` if loading `kernelbase.dll` fails for whatever reason.

> but at least we should call DelAttr to remove the functions, since the function pointers will be null.

Oops, my code was wrong. Fixed.